.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/{{ORG_NAME}}/{{REPO_NAME}}
```

Comment:
I think you want to always make this `linuxkit/linuxkit` as if I have forked it the gopath should not change

Reply:
Oh yeah, I had that and then read the docs which uses this and mislead me!